src/main/java/racingcar/CarInfo.java

```diff
@@ -0,0 +1,23 @@
+package racingcar;
+
+import lombok.Data;
+
+@Data
+public class CarInfo {
+    private int prevDistance = 0;
+    private int currentDistance = 0;
+    private int resultOfRand;
+
+    public void moveFoward() {
+        prevDistance = currentDistance;
+        currentDistance = prevDistance + 1;
+    }
+
+    public boolean isMoved() {
+        return currentDistance - prevDistance > 0;
+    }
+
+    public boolean isRandGreaterThan4() {
```

Comment:
```suggestion
    public boolean isRandGreaterThan(...)
```
세부사항을 외부로 과도하게 노출시키고 있는것 같아요

네이밍을 통해 세부사항을 과도하게 노출하는 것은 안좋다고 생각해요.
구현코드를 수정하면 `isRandGreaterThan4` 이름또한 수정해줘야 하는 부분이 있기 때문이에요.
이름을 좀 더 일반화 하거나,
변하는 부분을 변수로 분리해보아요.

Reply:
@wooobo 
말씀하신 문제점이 공감이 가네요
isRandGreaterThan(...)
파라미터를 받아 비교하도록 변경했습니다